crates/interpreter/src/interpreter.rs

```diff
@@ -96,14 +97,18 @@ impl Interpreter {
             instruction_result: InstructionResult::Continue,
             is_static,
             return_data_buffer: Bytes::new(),
-            shared_memory: EMPTY_SHARED_MEMORY,
-            stack: Stack::new(),
+            shared_context: EMPTY_SHARED_CONTEXT,
```

Comment:
shouldn't this be `::new()` to pre-allocate?

Reply:
Actually I don't think so, because we want to manually give the created context to then interpreter when it calls the `run` method of `EVMImpl`. See https://github.com/bluealloy/revm/pull/816/files/1de2425fed99f33422e0fa37911f7695a2a39c5b#diff-1d478ba44ccc56e3b1142bd3723bf97f3e254c25dd18323481aedadce0803e91R165-R170. Maybe I am missing something from what you said